lib/binaries/standalone_xml.ts

```diff
@@ -40,6 +40,9 @@ export class StandaloneXml extends XmlConfigSource {
       let latest = '';
       let latestVersion = '';
       for (let item of list) {
+        if (!item.endsWith('.jar')) {
```

Comment:
instead of this, maybe filter the list?

let jarList = list.filter((i) => i.endsWith('.jar');

Reply:
Sure?  It doesn't matter to me since this is from the legacy branch. Is this for efficiency or readability?

Reply:
Done.